packages/plugins/graphql/server/src/services/builders/type.ts

```diff
@@ -143,12 +146,37 @@ export default (context: Context) => {
       strapi,
     });
 
-    const args = attribute.multiple ? getContentTypeArgs(fileContentType) : undefined;
-    const type = attribute.multiple
-      ? naming.getRelationResponseCollectionName(fileContentType)
-      : naming.getEntityResponseName(fileContentType);
+    const args = attribute.multiple
+      ? getContentTypeArgs(fileContentType, { isNested: true })
+      : undefined;
+
+    const typeName = naming.getTypeName(fileContentType);
+
+    if (attribute.multiple) {
```

Comment:
If I understand this correctly, it means that for multiple media (and I guess toMany relations are the same), we'll always have two fields: rel and rel_connection, each with a different format, right? If yes, what about compatibility issues (conflicting names)?

Reply:
We should not have any conflicts as we use `_` in the connection suffix and all the other namings have to be camelCase